docs/idea-import.md

```diff
@@ -5,7 +5,7 @@ Most Bisq contributors use IDEA for development. The following instructions have
  1. Follow the instructions in [build.md](build.md) to clone and build Bisq at the command line.
  1. Open IDEA
  1. Go to `Help->Edit Custom Properties...`, add a line to the file that reads `idea.max.intellisense.filesize=12500` (to handle Bisq's very large generated `PB.java` Protobuf source file)
- 1. Go to `Preferences->Plugins`. Search for and install the _Lombok_ plugin. When prompted, do not restart IDEA.
+ 1. Go to `Preferences->Plugins` (`File->Settings, Plugins for Windows`). Search for and install the _Lombok_ plugin. When prompted, do not restart IDEA.
```

Comment:
@ripcurlx is `Preferences->Plugins` still valid for IntelliJ on MacOS? Because for both Windows and Ubuntu `File->Settings, Plugins` is valid.

Reply:
> @ripcurlx is `Preferences->Plugins` still valid for IntelliJ on MacOS? Because for both Windows and Ubuntu `File->Settings, Plugins` is valid.

It is still like that on MacOS